src/test/utils/SecurityUtils.test.tsx

```diff
@@ -2,7 +2,7 @@ import { decrypt, encrypt } from "../../utils/SecurityUtils";
 
 describe("Security Utils Tests", () => {
   it("Encrypt and Decrypt Test", () => {
-    const text = JSON.stringify({ data: "hello world" });
```

Comment:
Does this (and other changes to SecurityUtils) fix the flakiness?

Reply:
Yeah 🤞 not sure why I had it like this in the first place.